File: go/utility/make_context.go
package utility

import "github.com/voxgig-sdk/berufsbildung-daten-exporte-sdk/core"

func makeContextUtil(ctxmap map[string]any, basectx *core.Context) *core.Context {
	return core.NewContext(ctxmap, basectx)
}

File: py/berufsbildungdatenexporte_sdk/utility/make_context.py
# BerufsbildungDatenExporte SDK utility: make_context

from berufsbildungdatenexporte_sdk.core.context import BerufsbildungDatenExporteContext


def make_context_util(ctxmap, basectx):
    return BerufsbildungDatenExporteContext(ctxmap, basectx)
